errai-ioc/src/main/java/org/jboss/errai/ioc/Container.gwt.xml

```diff
@@ -22,6 +22,7 @@
   <inherits name="org.jboss.errai.common.Logging"/>
   <inherits name="javax.enterprise.Support"/>
   <inherits name="com.google.common.collect.Collect"/>
+  <inherits name="elemental2.dom.Dom"/>
```

Comment:
After removing the `uncheckedCast`s from `org.jboss.errai.ioc.rebind.ioc.bootstrapper.TypeFactoryBodyGenerator` this should've been removed as well. Changing in a bit.